### PR TITLE
feat: Getting layout setting via rest api

### PR DESCRIPTION
**Pull request by andpopov**
@olenagerasimova Please review

### CODE REVIEW

File: src/main/java/com/artipie/api/SettingsRest.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+/**
+ * REST API methods to manage Artipie settings.
+ * @since 0.27
+ */
+public final class SettingsRest extends BaseRest {
+    /**
+     * Settings.
+     */
+    private final Settings settings;
```
Comment (olenagerasimova) [inline src/main/java/com/artipie/api/SettingsRest.java:22]: Why do we have `Settings` here if the only thing we is layout?

File: src/main/java/com/artipie/api/RestApi.java [hunk 2]
```diff
@@ -71,6 +72,11 @@ public final class RestApi extends AbstractVerticle {
      */
     private final Optional<KeyStore> keystore;
 
+    /**
+     * Artipie settings.
+     */
+    private final Settings settings;
```
Comment (olenagerasimova) [inline src/main/java/com/artipie/api/RestApi.java:78]: I do not think we should have `Settings` as a field here, as it's not really required, layout is already in the class fields

File: src/main/java/com/artipie/VertxMain.java [hunk 3]
```diff
@@ -115,7 +115,7 @@ public int start(final int apiport) throws IOException {
             auth -> vertx.deployVerticle(
                 new RestApi(
                     caches, settings.repoConfigsStorage(), settings.layout().toString(),
-                    apiport, settings.credentialsKey(), auth, settings.keyStore()
+                    apiport, settings.credentialsKey(), auth, settings.keyStore(), settings
```
Comment (olenagerasimova) [inline src/main/java/com/artipie/VertxMain.java:118]: and this looks strange, as we first ask credentials key and key store from the settings and then pass the whole settings instance...

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@andpopov thanks, I think we can simply pass layout to `SettingsRest` instead of whole `Settings` instance

File: src/main/java/com/artipie/VertxMain.java (hunk 3 above)
Comment (andpopov) [reply src/main/java/com/artipie/VertxMain.java]: @olenagerasimova Thanks, I have fixed, please review

**Review by olenagerasimova — APPROVED** [review]
@andpopov thanks

**Pull request merged**